### DISCUSSION TITLE
Import .cer teleport to AD get error

### DISCUSSION THREAD

**Discussion by ducth92** [gravitational/teleport#27656] [Q&A] [answered]
<img width="1082" alt="image" src="https://github.com/gravitational/teleport/assets/78575567/76f2b9bc-d750-4334-8d00-3ba57be97831">

Follow the guide at this link: https://goteleport.com/docs/desktop-access/active-directory-manual/
At Step 3/7. Configure a GPO to allow Teleport connections , 
- On AD: when I choose the file .cer which export from teleport server, It get the isse "the file type is note recognizable. Select another file." but on the video in guide, they can do it. Have anyone met that issue? Any how can you fix it? Thanks all!

**Comment by webvictim** [comment, marked as answer]
This is an error in the docs, my apologies. `curl` as an alias to `Invoke-WebRequest` on Windows outputs a lot of garbage and the command is incorrect.

Use this command instead (replacing `teleport.example.com` with the URL of your Teleport server):
```
curl -o user-ca.cer https://teleport.example.com/webapi/auth/export?type=windows
```

**Reply by webvictim** [reply]
I've submitted a PR to fix the docs: https://github.com/gravitational/teleport/pull/27734

Thanks for your post!
